OneSignalSDK/unittest/src/test/java/com/test/onesignal/RESTClientRunner.java

```diff
@@ -67,7 +78,10 @@ public static void setUpClass() throws Exception {
    @Before // Before each test
    public void beforeEachTest() throws Exception {
       firstResponse = secondResponse = null;
+      blankActivityController = Robolectric.buildActivity(BlankActivity.class).create();
+      blankActivity = blankActivityController.get();
       TestHelpers.beforeTestInitAndCleanup();
+      OneSignal.setAppContext(blankActivity);
```

Comment:
`ApplicationProvider.getApplicationContext()` is okay to set context with also and it will be more consistent with our other UnitTests. However, I think we should call this inside of the UnitTests individually so its easier to understand that this is an important part of the new OneSignal init